backend/lib/db.ts

```diff
@@ -1,3 +1,90 @@
+import { join } from 'node:path';
 import postgres from 'postgres';
 
-export const sql = postgres(process.env.DATABASE_URL);
+import { fileExtension, uuidv4 } from './utils';
+
+export const sql = postgres(process.env.DATABASE_URL || '');
+
+export async function uploadFile(
+    file: File,
+    uploadById: string,
+    options: {
```

Comment:
what is public true? like anyone can access the file if `options: { public : true }` ?

Reply:
Default as public, yes. For false and other value, I'm going to implement access control later when needed